### DISCUSSION TITLE
2FA

### DISCUSSION THREAD

**Discussion by CostGamer** [community/community#143939] [Other Feature Feedback, Questions, & Ideas]
Plz tell me how to find 2FA code. I even couldn't send a message to support without this code. I am sure that I adjusted everything properly but I couldn't find a code

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @CostGamer ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
